### PR TITLE
feat: Volunteers Section

### PR DESCRIPTION
**Pull request by 10lite**
disclaimers:
- bruteforced code?
- scuffed var names
- refactorable
- buttt I think it works. ideal if ma-test with complete cms data

### CODE REVIEW

File: src/components/widgets/volunteers/Volunteers.astro [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  classes?: { container?: string };
+  bg?: any;
+}
+
+const { id, volunteersInfo } = Astro.props;
+
+// Committee order (from Figma)
+const committeeOrder = ['Creatives', 'Communications', 'Programs', 'Promotions', 'Logistics', 'Technicals', 'Website'];
+
+const groupedVolunteers = committeeOrder.map(committee => ({
```
Comment (seangaaab) [inline src/components/widgets/volunteers/Volunteers.astro:21]: Nitpick: Rename `groupedVolunteers` to `committees`

File: src/components/widgets/volunteers/Volunteers.astro [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  title: committee,
+  members: volunteersInfo.filter(volunteer => volunteer.committee.title === committee)
+}));
+
+// For volunteer name consistency
+const getFirstName = (name: string) => {
```
Comment (seangaaab) [inline src/components/widgets/volunteers/Volunteers.astro:27]: Move the `getFirstName` and `getLastName` into `src/utils/utils.ts`

File: src/pages/index.astro [hunk 3]
```diff
@@ -21,6 +22,7 @@ const metadata = {
 };
 
 const cmsData = await fetchCMS();
+console.log(cmsData.volunteers);
```
Comment (seangaaab) [inline src/pages/index.astro:25]: Remove the `console.log`

**Review by seangaaab — CHANGES_REQUESTED** [review]
Left some few comments for improvements. 

Other than that, LGTM! 

Good job! Thanks!

**Comment by 10lite** [comment]
Addressed the improvements in the latest commit, thanks!

**Pull request merged**